README.md

```diff
@@ -18,6 +18,7 @@ A Charmed Operator for SD-Core's gNodeB simulator (GNBSIM) component.
 
 ```bash
 juju deploy sdcore-gnbsim --trust --channel=edge
+juju run sdcore-gnbsim/leader start-simulation
```

Comment:
Do we expect this to be run in the same model (thus, same namespace) as the rest of sdcore?

Maybe we should add some details on how to find the right configuration.

Reply:
The specific namespace/model doesn't matter so much, as long as the AMF and UPF addresses are reachable from the gnbsim. And yes we'll have to document better how to run the simulation alongside the sdcore bundles. My goal was to add this in the readthedocs documentation because of the interdependence of configuration between the 2.